homa.h

```diff
@@ -55,9 +55,9 @@ extern "C"
  * sockaddr_storage).
  */
 typedef union sockaddr_in_union {
-        struct sockaddr sa;
-        struct sockaddr_in in4;
-        struct sockaddr_in6 in6;
+	struct sockaddr sa;
```

Comment:
This commit replaces whitespaces with tabs. This goes against both Linux and my personal coding conventions and is really not a good idea because tabs are treated differently by different IDEs and editors. I've been burned by tabs too many times in the past, so I'm not going to apply this commit.

Reply:
Can we define what the expected indentation style for this project is? There are 5607 lines that start with tab in the main directory, vs 1426 that start with spaces. It's all very confusing.

Reply:
Oops, my bad! I should have checked my IDE settings before responding. The Linux kernel style is actually to use tabs (!), so this commit does make sense. Sorry about that.